tests/app/validation/test_date_range_validator.py

```diff
@@ -16,17 +16,17 @@ def test_date_range_matching_dates(self):
 
         validator = DateRangeCheck()
 
-        period_from = Mock()
+        period_from = MagicMock()
```

Comment:
@MebinAbraham just out of interest why are we using `MagicMock` instead of `Mock`